packages/mwp-app-route-plugin/src/handler.js

```diff
@@ -20,7 +21,7 @@ export default (languageRenderers: { [string]: LanguageRenderer }): HapiHandler
 ) => {
 	const pathname = request.getLangPrefixPath();
 	if (pathname !== request.url.pathname) {
-		return h.redirect({ ...request.url, pathname }.toString());
```

Comment:
this is fine, but I think it might be a bit cleaner to just use the `URL` interface and modify `pathname` on a new copy, e.g.

```js
const newUrl = new URL(request.url); // make a copy
newUrl.pathname = pathname;
return h.redirect(newUrl.toString());
```